kong/pdk/client/tls.lua

```diff
@@ -1,6 +1,6 @@
 ---
--- The client.tls module provides functions for interacting with TLS
--- connections from client.
+-- A module that provides functions for interacting with TLS
```

Comment:
Nit: I think the "A module that..." phrasing feels a little inconsistent with the doc strings on other PDK modules, which seem to (quite loosely) follow this kind of template:

```
---
-- {{ title }} (e.g. "Client TLS module")
--
-- {{ description sentence/fragment: what it does and/or what it contains }} (e.g. "Functions for interacting with client TLS connections." or similar)
-- @module {{ module name }}
```

Examples:
* [kong.pdk.response](https://github.com/Kong/kong/blob/86439965e5fe8101c6a468d5674710a9d0f785b4/kong/pdk/response.lua#L1-L7)
* [kong.pdk.client](https://github.com/Kong/kong/blob/86439965e5fe8101c6a468d5674710a9d0f785b4/kong/pdk/client.lua#L1-L7)
* [kong.pdk.router](https://github.com/Kong/kong/blob/86439965e5fe8101c6a468d5674710a9d0f785b4/kong/pdk/router.lua#L1-L4)